pkg/util/util_test.go

```diff
@@ -127,3 +127,17 @@ func TestBoolPtrIsNilOrFalse(t *testing.T) {
 	assert.True(t, BoolPtrIsNilOrFalse(&fb))
 	assert.False(t, BoolPtrIsNilOrFalse(&tb))
 }
+
+func TestGetIndexFromSlice(t *testing.T) {
```

Comment:
👍 for fun names. 

This is not strictly necessary but worth mentioning. You _could_ use a  [test table](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) here.

```go
	testcases := []struct {
		idx    int
		slice  []string
		elem string
	}{
		{0, []string{"Penn", "Teller"}, "Penn"},
		// ...
	}
	for _, tc := range testcases {
		assert.EqualValues(t, tc.idx, GetIndexFromSlice(tc.slice, tc.elem))
	}
```

Reply:
I tend to add them or their equivalent (JUnit Parameterized Tests, Testng Data Provider) when the tests cant be succinctly expressed (like on one line)